docs/application/native/guides/ui/efl/wearable/component-circle-rotary.md

```diff
@@ -95,8 +95,8 @@ The following table shows the configurable rotary selector parts.
 |----------------------------------------|----------------------------------------|----------------------------------------|
 | `selector,main_text`                     | `eext_rotary_selector_item_part_color_set()` | ![main_text](./media/rotary_selector_main.png) |
```

Comment:
Broken images in this chart. It looks like images are in wrong folder. They are in `../media/` not in `./media/`.

Reply:
done